### ISSUE TITLE
Strange issue console Mac OSX

### ISSUE THREAD

**Issue by Maximilian-Schwarz** [vuestorefront/shopware-pwa#1781]
### What happened?

I want to init a new project and i got a very strange view in the terminal. I run the following command:
`npx @shopware-pwa/cli init`

Any ideas?

![Bildschirmfoto 2022-01-10 um 16 12 35](https://user-images.githubusercontent.com/63093230/148789413-490fb95e-c48e-4444-a41e-fedde3adfe46.png)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

Latest stable (generated by CLI)

### Environment

```markdown
- OS: macOS
- Node: v14.16.0
```

**Comment by mkucmus** [comment]
hi @Maximilian-Schwarz, thanks!
One of our dependency has a "buggy" dependency and we are working on it 👍🏻

**Comment by mkucmus** [comment]
you can try this one:
1. run `yarn add -D @shopware-pwa/cli@canary` in an empty directory 
2. open and edit `package.json` to add: 
```
  "resolutions": {
    "colors": "<=1.4.0"
  }
 ```
 3. run `yarn shopware-pwa init`

**Comment by mkucmus** [comment]
fixed via #1780
